common/app/views/support/cleaner/VideoEmbedCleaner.scala

```diff
@@ -8,7 +8,7 @@ import views.support.{HtmlCleaner, Item640}
 
 import scala.collection.JavaConversions._
 
-case class VideoEmbedCleaner(article: Article) extends HtmlCleaner {
+case class VideoEmbedCleaner(article: Article, maxEmbedHeight: Int = 680) extends HtmlCleaner {
```

Comment:
@akash1810 do you know if applying this max height of 680px on embedded video iframes is OK?

Reply:
we'd probably want this to be `812px` for fullscreen on iPhoneX? https://mydevice.io/devices/

Reply:
Makes sense, I've pushed that up now.